extensions/amp-sidebar/1.0/amp-sidebar.js

```diff
@@ -18,13 +18,27 @@ import {BaseElement} from './base-element';
 import {CSS} from '../../../build/amp-sidebar-1.0.css';
 import {isExperimentOn} from '#experiments';
 import {userAssert} from '../../../src/log';
+import {Services} from '#service/';
 
 /** @const {string} */
 const TAG = 'amp-sidebar';
 
 class AmpSidebar extends BaseElement {
+  /** @override */
+  constructor(element) {
+    super(element);
+
+    /** @private {!../../../src/service/history-impl.History} */
+    this.history_ = null;
+
+    /** @private {number} */
```

Comment:
nit: I wonder if this would be more clear as a `number|null`, instead of coopting `-1` to mean "no history"

Reply:
Done